test/command_line/helper/command_runner.rb

```diff
@@ -182,7 +182,34 @@ def normalize_init_line(line)
     end
   end
 
+  def normalize_groonga_log(text)
+    normalized = ""
+    text.split("\n").each do |line|
+      line = normalize_init_line(line)
+      if line =~ /grn_init: <VERSION>/ or
+        line =~ /grn_fin / or
+        line =~ /vm.overcommit_memory / or
+        line =~ /spec:\d+/ or
+        line =~ /DDL:\d+/ or
+        line =~ /To set vm.overcommit_memory/ or
+        line =~ /run 'sudo \/sbin\/sysctl/
+      else
+        next if stack_trace?(line)
+        # normalize temporary file generated by grn_mkstemp
+        replaced = line.gsub(/: <(.+\.db\.\d{7})[0-9a-zA-Z]{6}>/) do |path|
+          ": <#{$1}XXXXXX>"
+        end
+        normalized << "#{replaced}\n"
+      end
+    end
+    normalized
+  end
+
   private
+  def stack_trace?(line)
+    line =~ /\|e\| (.+?)\((.+?)\) \[(0x.+?)\]$/
```

Comment:
Ruby is object oriented language. We should use `Regexp` method instead of `String` method to match by regular expression.

BTW, `/.../.match?(string)` is better because we just need to know whether matched or not.

Reply:
fixed.